modules/ROOT/pages/_partials/generic-local-mobile-dev-envir-reqs.adoc

```diff
@@ -8,7 +8,7 @@ You need the following installed on your machine:
 * [x] Android SDK or Android Studio from https://developer.android.com/studio/index.html[Google, window="_blank"]
 * [x] (Recommended) The latest version of http://openjdk.java.net/install/index.html[OpenJDK, window="_blank"]
 
-For information on how to set up your local development environment for Android, see link:https://developer.android.com/studio/install[Android, window="_blank"].
+For information on how to set up a local Android development environment, see link:https://developer.android.com/studio/install[Android, window="_blank"].
```

Comment:
see the link:https://developer.android.com/studio/install[Android Studio documentation, window="_blank"].